DOC.md

```diff
@@ -629,6 +629,35 @@ Note load vscode-style packages using `require("luasnip.loaders.from_vscode").lo
 you've configured luasnip to detect the filetype based on the cursor position. Else the
 snippets won't be available to the `from_cursor_pos` function.
 
+# SNIPMATE SNIPPETS LOADER
+
+As luasnip is capable of loading the same format of vim-snipmate, It can be used
+in the same way as `from_vscode`.
+
+```lua
+ 	require("luasnip/loaders/from_snipmate").load(opts) -- opts can be ommited
+```
+
+See `from_vscode` for an explanation of opt.
+If opts.paths be ommited, "snippets" folders in rtp are used
+
+Luasnip is compatible with [honza/vim-snippets](https://github.com/honza/vim-snippets).
+Please use it as a reference for your directory structure.
+
+Lazy loading is also available here.
+
+```lua
+ 	require("luasnip.loaders.from_snipmate").lazy_load(opts) -- opts can be ommited
```

Comment:
You're missing the `filetype_extend` :P

Reply:
Marked the wrong code-block, could you put that into the one with the normal `load()`?

Reply:
Ah, just saw that you added it to `Examples/snippets.lua`, I think it would make sense to also include here.

Reply:
It's just a note about using honza/vim-snippets, and isn't it better here?